releasenotes/config.yaml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
these probably aren't the final config options we want, I was just messing about.

Reply:
@brettlangdon  Are these the final config options that we want? 
Also, what does the stop_at_branch_base option do?

Reply:
probably not, I removed for now since in `master` we don't have a config defined yet